runtimes/eden/src/version.rs

```diff
@@ -40,7 +40,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	/// Version of the runtime specification. A full-node will not attempt to use its native
 	/// runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
 	/// `spec_version` and `authoring_version` are the same between Wasm and native.
-	spec_version: 24,
+	spec_version: 25,
```

Comment:
```suggestion
	spec_version: 24,
```

Current spec is 23 no need to go full 25 yet